doc/ledger3.texi

```diff
@@ -4616,13 +4616,42 @@ from the file above.
 @findex --rich-data
 
 The @command{convert} command accepts three options. They are
-@option{--invert} which inverts the amount field,
-@option{--account @var{STR}} which you can use to specify the account to
-balance against, and @option{--rich-data} which stores
-additional metadata as tags. There is, for example,
-a UUID field. If an entry with the same UUID tag is already included in
-the normal ledger file (specified via @option{--file @var{FILE} (-f)} or
-via the environment variable @env{LEDGER_FILE}) this entry will not be
+@option{--invert} which inverts the amount field, @option{--account
+@var{STR}} which you can use to specify the account to balance against,
+and @option{--rich-data} which stores additional metadata as tags.
+
+Using following command on the two first lines of the above csv file,
+
+@smallexample
```

Comment:
Unfortunately no, as DocTests.py is not able to handle multi-lines ledger commands. I copied-pasted an already existing comment to track it as TODO.